package.json

```diff
@@ -53,9 +53,10 @@
     "eslint-plugin-prettier": "3.1.1",
     "mocha": "6.2.2",
     "nyc": "14.1.1",
+    "postcss": "^8.0.0",
     "postcss-scss": "2.0.0"
   },
-  "dependencies": {
-    "postcss": "^7.0.0"
+  "peerDependencies": {
+    "postcss": "^8.0.0"
   }
```

Comment:
Last time I checked peer dependencies are not installed when doing:
```
$ npm install postcss-selector-namespace
```

So I think we might need to update the readme that `postcss` has to be installed as well?

```
$ npm install postcss postcss-selector-namespace
```

what do you think?

Reply:
I think that is a good idea.

> npm versions 1 and 2 will automatically install peerDependencies if they are not explicitly depended upon higher in the dependency tree. For all following versions of npm (starting with npm@3), this will no longer be the case. You will receive a warning that the peerDependency is not installed instead.
> https://nodejs.org/en/blog/npm/peer-dependencies/